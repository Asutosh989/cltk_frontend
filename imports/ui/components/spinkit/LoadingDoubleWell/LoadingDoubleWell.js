export default class LoadingDoubleWell extends React.Component {
	render() {
		return (
			<div className="loading-spinner loading-spinner-double-well">
				<div className="double-bounce1"></div>
				<div className="double-bounce2"></div>
			</div>
		);
	}
}
