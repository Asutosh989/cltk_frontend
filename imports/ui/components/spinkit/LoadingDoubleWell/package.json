{
  "name": "LoadingDoubleWell",
  "version": "0.0.1",
  "private": true,
  "main": "./LoadingDoubleWell.js"
}
