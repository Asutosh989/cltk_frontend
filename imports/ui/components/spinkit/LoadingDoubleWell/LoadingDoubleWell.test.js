
import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import LoadingDoubleWell from './LoadingDoubleWell';

describe('LoadingDoubleWell', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(
				<MuiThemeProvider>
					<LoadingDoubleWell />
				</MuiThemeProvider>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
