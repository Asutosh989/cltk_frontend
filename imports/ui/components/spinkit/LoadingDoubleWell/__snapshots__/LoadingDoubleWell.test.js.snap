// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingDoubleWell renders correctly 1`] = `
<div
  className="loading-spinner loading-spinner-double-well"
>
  <div
    className="double-bounce1"
  />
  <div
    className="double-bounce2"
  />
</div>
`;
